pkg/maven/settings.go

```diff
@@ -169,11 +169,16 @@ func CreateNewProjectSettingsXML(altDeploymentRepositoryID string, altDeployment
 
 func UpdateProjectSettingsXML(projectSettingsFile string, altDeploymentRepositoryID string, altDeploymentRepositoryUser string, altDeploymentRepositoryPassword string, utils SettingsDownloadUtils) (string, error) {
 	projectSettingsFileDestination := ".pipeline/mavenProjectSettings"
```

Comment:
does `.pipeline/mavenProjectSettings` always exist? A user can have a custom file, if this is not the case `.pipeline/mavenProjectSettings` is used, right?